inspect-traffic/inspector-insights.md

```diff
@@ -197,6 +198,11 @@ text/x-javascript
 
 ![JavaScript Inspector](../images/livetraffic/inspectors/inspectors-javascript.png)
 
+### Auth Inspector
+
+The **Auth inspector** displays authentication-related tokens and data, such as JSON Web Token (JWT), Bearer token (Basic Auth), and Security Assertion Markup Language (SAML) assertions. The **Auth** inspector is available for live traffic sessions and saved snapshots.
```

Comment:
Bearer token is not Basic Auth. Basic authentication is it's own thing.
More often than not, JWT Tokens are used as Bearer tokens.
What I'd say about the Auth tab currently is, it obtains authorization information from Authorize and Proxy-Authorize headers, and obtains SAML requests and responses from the Query Params and the Request body, then tries to decode this information if possible (for auth headers, we only support Basic auth scheme and JWT Bearer tokens)